utils/logging/log.go

```diff
@@ -369,23 +389,29 @@ func (fw *fileWriter) Rotate() error {
 	return nil
 }
 
-func (fw *fileWriter) create(fileIndex int) (*bufio.Writer, *os.File, error) {
-	filename := filepath.Join(fw.config.Directory, fmt.Sprintf("%d.log", fw.fileIndex))
-	file, err := os.Create(filename)
+// Creates a file if it does not exist or opens it in append mode if it does
+func (fw *fileWriter) create() (*bufio.Writer, *os.File, error) {
+	filename := filepath.Join(fw.config.Directory, fmt.Sprintf("%s.log", fw.config.LoggerName))
```

Comment:
Shouldn't actually this start with `%s.0.log`? Will rotation work otherwise?

Reply:
Rotated logs start at 1 and the main log is written to [log_filename].log with no extension (again going by logrotate's default naming conventions).
So your log folder would look something like this:
> C.http.log                  <= the current log file at all times
> C.http.log.1               <= most recently rotated log file 
> C.http.log.2               <= second most recently rotated log file
> C.http.log.3
> ...